### PR TITLE
Made anime info search better

### PR DESCRIPTION
**Pull request by Yureien**
Now, offline search is available.
~Before searching for info with aniDB, run the `init_search.py` script (See the README for a detailed how-to)~
DONT DO ABOVE.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ Anime scrapers is a collection of scrapers that have been all unified.
...
 ## Usage
 
+**IMPORTANT:** Before using the `info_handler` module, please run `init_search.py` (recommended to run every two weeks. Run at MAXIMUM once per day.)
+To run it, use this command:
+```
+cd <To the anime_scrapers directory> OR OPEN THE TERMINAL IN THE anime_scrapers DIRECTORY.
+python3
+# The Python terminal will run, type/copy-paste everything after '>>> ' below.
+>>> from init_search import download_list
+>>> download_list.download()
```
Comment (jQwotos) [inline README.md]: There is no download function within the DownloadList class in init_search.py

**Review by jQwotos — CHANGES_REQUESTED** [review]

**Comment by Yureien** [comment]
@jQwotos Fixed

**Comment by jQwotos** [comment]
I'm getting this error
```
>>> from init_search import download_list
>>> download_list.get_file()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jasonle/Downloads/anime_scrapers/init_search.py", line 55, in get_file
    self.untar_list()
  File "/Users/jasonle/Downloads/anime_scrapers/init_search.py", line 49, in untar_list
    shutil.copyfileobj(f_in, f_out)
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/shutil.py", line 79, in copyfileobj
    buf = fsrc.read(length)
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/gzip.py", line 276, in read
    return self._buffer.read(size)
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/gzip.py", line 463, in read
    if not self._read_gzip_header():
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/gzip.py", line 411, in _read_gzip_header
    raise OSError('Not a gzipped file (%r)' % magic)
OSError: Not a gzipped file (b'<?')
>>> 
```
And the `anime-title.xml` file is blank.
Which causes a search to fail with a None error since it's blank.

---

Also, I don't think you need to unpack anime-titles.xml.gz (looking at it you should be able to just rename it)

Seems to work when I manually renamed it as well.
```
>>> from anime_scrapers.info_handler import info_handler as ih
>>> ih.search('log horizon')
[[{'id': 9677, 'titles': ['ログ・ホライズン', 'Log Horizon', 'Log Horizon', 'Log Horizon', 'ورود به افق', '记录的地平线']}]]
>>> 
```

**Review by jQwotos — APPROVED** [review]

**Comment by Yureien** [comment]
@jQwotos Fixed the bug

**Comment by Yureien** [comment]
@jQwotos Changed how it works, now, no need to run `init_search.py`. It happens automatically during search.

**Pull request merged**